.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     name: Website build
     runs-on: ubuntu-latest
     needs: pre_check
-    if: ${{ needs.pre_check.outputs.should_skip_website != 'true' }}
+    if: ${{ needs.pre_check.outputs.should_skip_website != 'true' }} || ${{ needs.pre_check.outputs.should_skip_lang != 'true' }}
```

Comment:
Should should_skip_components be checked here as well?

Reply:
If components need to be rebuilt then so does the website?

Reply:
`website` is not dependent on `components`.

Reply:
nope netlify is par with ci.

Reply:
Website is dependent on components?

Reply:
I think what's happening now is netlify and ci for `website` are each pulling `@quri/squiggle-components` into `node_modules` at the monorepo `yarn` command (i.e. from `npmjs.com` registry) and not from the repo's code.

Reply:
Ok so 
```yml
    - uses: actions/checkout@v2
    - name: Install dependencies from monorepo level
      run: cd ../../ && yarn
    - name: Build rescript in squiggle-lang
      run: cd ../squiggle-lang && yarn build
    - name: Build website assets
      run: yarn build
```
should also have `cd ../components && yarn build` thrown in as an extra step before building website assets?

Reply:
No....

Reply:
resolved on video call.